### DISCUSSION TITLE
How to set `drivers_home` variable from command-line?

### DISCUSSION THREAD

**Discussion by fdcastel** [dbeaver/dbeaver#29332] [Q&A]
**How can I set `${drivers_home}` variable from `dbeaver.ini`?**

I want to put drivers out of my workspace folder (e.g. in `C:\drivers`).

I'm already using `-Ddbeaver.drivers.configuration-file=C:\drivers\drivers.xml`. And this is working fine.

However... `drivers.xml` still uses a `${drivers_home}` variable. And it still points to my user workspace.

From menu _Help / Installation Information / Configuration_ I got this:

```
DRIVERS_LOCATION=C:\Users\administrator\AppData\Roaming\DBeaverData\drivers
```

But everything I tried until now to change this value was unsuccessful:

```
-Ddbeaver.drivers.location=C:\drivers
-Ddbeaver.drivers.home=C:\drivers
-DDRIVERS_LOCATION=C:\drivers
```

> _Using:_
> DBeaver version 24.0.3.202404211624
> Windows Server 2022 Standard (version 10.0.20348.1850)

**Comment by fdcastel** [comment]
Alternatively, is there a way to avoid this dialog and just make DBeaver download the driver automatically?

![image](https://github.com/dbeaver/dbeaver/assets/629408/51e1e85d-8efe-42f3-b514-45d28a07af15)

**Comment by LonwoLonwo** [comment]
Hello

Did you try to use this settings section?
![image](https://github.com/dbeaver/dbeaver/assets/45152336/97fc2504-802d-4208-8c10-b0de98528a1d)

**Reply by fdcastel** [reply]
Sorry @LonwoLonwo I don't understand what you are suggesting me to do. 

I want to set `${drivers_home}` value (a template value used inside `drivers.xml`) from the command-line.

Something like 

```
dbeaver.exe -Ddrivers_home=C:\drivers
```

**Reply by superwinni2** [reply]
I'm trying the same thing.
I've got 2 PCs with different usernames. I sync files between them in Userdirectory\filesync.

I've set on PC1 with user1 the default driver path to C:\users\user1\filesync\drivers.
But on PC2 the path should be C:\users\user2\filesync\drivers
I also tried to use something like a systemvariable %USERPROFILE% but that didn't work.

I found the config file in the Workspace\\.metadata\\.plugins\\org.eclipse.core.runtime\\.settings\\org.jkiss.dbeaver.core.prefs file.
There are 2 entries:

- dialog.default.folder
- ui.drivers.home

The second one sounds like the right one.
How can I do a local overwrite via command line?
